portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,71 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomATLAQuote() {
+  const quotes =
+      ['"Appa, yip yip!" - Aang', 
+      '"That\'s rough, buddy." - Prince Zuko', 
+      '"My own mother thought I was a monster. She was right of course, but it still hurt." - Princess Azula', 
+      '"When we hit our lowest point, we are open to the greatest change." - Aang', 
+      '"It is important to draw wisdom from different places. If you take it only from one place it becomes stale and rigid." - Uncle Iroh',
+      '"Love is brightest in the dark." - Katara',
+      '"My cabbages!" - Cabbage Merchant'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  // Pick a random quote.
+  const quote = quotes[Math.floor(Math.random() * quotes.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const quoteContainer = document.getElementById('quote-container');
+  quoteContainer.innerText = quote;
+
+}
+
+function addRandomATLAQuote() {
+  const quotes =
+      ['"Appa, yip yip!" - Aang', 
+      '"That\'s rough, buddy." - Prince Zuko', 
+      '"My own mother thought I was a monster. She was right of course, but it still hurt." - Princess Azula', 
+      '"When we hit our lowest point, we are open to the greatest change." - Aang', 
+      '"It is important to draw wisdom from different places. If you take it only from one place it becomes stale and rigid." - Uncle Iroh',
+      '"Love is brightest in the dark." - Katara',
+      '"My cabbages!" - Cabbage Merchant'];
+
+  // Pick a random quote.
+  const quote = quotes[Math.floor(Math.random() * quotes.length)];
+
+  // Add it to the page.
+  const quoteContainer = document.getElementById('quote-container');
+  quoteContainer.innerText = quote;
+
 }
+
+function addRandomMusicRecommendation(){
+    // Create arrays for song titles and their track images; both are related by indices
+    const songTitles = ['"telepatía" - Kali Uchis',
+                        '"She" - Harry Styles',
+                        '"Adiós" - Selena Gomez',
+                        '"Call My Friends" - Shawn Mendes',
+                        '"My Hair" - Ariana Grande'];
+    const songImages = ['/images/telepatia.jpg', 
+                        '/images/She.jpg',
+                        '/images/Adios.png',
+                        '/images/CallMyFriends.JPG',
+                        '/images/MyHair.JPG'];
```

Comment:
Thank you for the suggestion! I modified the function to hold an array of objects instead of the two separate arrays and added: the year the song was released, the album the song is from, and a link to the song on YouTube.